sdk/keyvault/azkeys/platform-matrix.json

```diff
@@ -0,0 +1,16 @@
+{
+    "displayNames": {
+        "@{ enableHsm = $true }": "HSM"
+    },
+    "include": [
+        {
+            "Agent": {
+                "ubuntu-20.04": {
+                    "OSVmImage": "MMSUbuntu20.04",
+                    "Pool": "azsdk-pool-mms-ubuntu-2004-general"
+                }
+            },
+            "ArmTemplateParameters": "@{ enableHsm = $true }"
```

Comment:
Based on the current failure you need to also set GoVersion: https://github.com/Azure/azure-sdk-for-go/blob/main/eng/pipelines/templates/stages/platform-matrix.json#L13